packages/sdk/deploy/deployLiquidator.ts

```diff
@@ -0,0 +1,34 @@
+import { DeployFunction } from "hardhat-deploy/types";
+
+import { ChainDeployConfig, chainDeployConfig } from "../chainDeploy";
+import { configureIonicLiquidator, deployIonicLiquidator } from "../chainDeploy/helpers/liquidators/ionicLiquidator";
+
+const func: DeployFunction = async ({ run, ethers, getNamedAccounts, deployments, getChainId }) => {
```

Comment:
If this is relevant to deployment we need to put this in the new deployment structure.